src/jobsautoreport/query.py

```diff
@@ -35,21 +36,73 @@ def _get_query_all_jobs(from_date: datetime, to_date: datetime) -> dict:
             }
         }
 
-    def query_all_jobs(
-        self, from_date: datetime, to_date: datetime
-    ) -> list[JobDetails]:
+    @staticmethod
+    def _get_query_all_steps(from_date: datetime, to_date: datetime) -> dict:
```

Comment:
```suggestion
    def _get_query_steps_by_name(from_date: datetime, to_date: datetime, name: str) -> dict[str, Any]:
```
could be good to add a parameter for the step name